src/boot/bootctl.c

```diff
@@ -586,7 +586,7 @@ static int print_efi_option(uint16_t id, int *n_printed, bool in_order) {
 
         r = efi_get_boot_option(id, &title, &partition, &path, &active);
         if (r < 0)
-                return log_error_errno(r, "Failed to read boot option 0x%04X: %m", id);
+                return log_debug_errno(r, "Failed to read boot option 0x%04X: %m", id);
```

Comment:
hmm, this should not propagate the error then I guess. i.e. return 0. Also say `, ignoring` as usual

Reply:
→ #25439